dangerfile.ts

```diff
@@ -0,0 +1,48 @@
+import { danger, message, warn, fail } from 'danger'
+
+const pr = danger.github.pr
+const isUser = pr.user.type === 'User'
+const files = danger.git.modified_files

```

Comment:
Destructuring?

Reply:
I think it will not be clear with destructuring